rpc/api/personal.go

```diff
@@ -98,9 +98,22 @@ func (self *personalApi) NewAccount(req *shared.Request) (interface{}, error) {
 	if err := self.codec.Decode(req.Params, &args); err != nil {
 		return nil, shared.NewDecodeParamError(err.Error())
 	}
-
+	var passwd string
+	if args.Passphrase == nil {
+		fe := self.xeth.Frontend()
+		if fe == nil {
+			return false, fmt.Errorf("unable to create account: password cannot be input interactively")
+		}
+		var ok bool
+		passwd, ok = fe.AskPassword()
+		if !ok {
+			return false, fmt.Errorf("unable to create account: password cannot be input interactively")
```

Comment:
The error messages on line 105 and 110 can be improved I think.
Something in the lines of:

L105: Unable to interact with user
L110: User didn't give password